backend/src/main/java/com/rota/auth/AuthenticationUtils.java

```diff
@@ -12,17 +13,37 @@
    * @return Whether authentication was successful.
    */
   public static boolean validateToken(String token) {
-    return true;
+    return !token.toLowerCase().equals("false");
   }
 
   /**
    * Extracts user information from the provided token.
    * @param token Authentication token.
    * @return User information if token is valid and if present.
    */
-  public static Optional<String> getUserFromToken(String token) {
+  public static Optional<Integer> getUserFromToken(String token) {
+    Integer parsedToken;
+    try {
+      parsedToken = Integer.parseInt(token);
+    } catch (NumberFormatException e) {
+      return Optional.empty();
+    }
+    return Optional.of(parsedToken);
+  }
+
+  /**
+   * TODO Temp solution needs to be changed to be more appropriate when auth is added to backend.
+   * Get user role from token.
+   * @param token Authentication token
+   * @return {@link Role} of the user or <code>null</code> if token is not valid.
+   */
+  public static Optional<Role> getUserRoleFromToken(String token) {
     return validateToken(token)
-        ? Optional.of(token)
+        ? Optional.of(
+          (token.length() % 2 == 0)
```

Comment:
incase we add additional roles later on, would you be better checking if the list contains Role.Manager?

Reply:
Wasn't sure how we're going to manage roles, and didn't want to just take a decision here in case it makes it harder later. Need to decide if it'll depend on using Auth0 roles (and setting them up) or based off of the roles as in our db. Also we might have to somehow synchronise the 2. 

This was just a placeholder and should be changed once we decide that.

Reply:
Sure. Let's discuss roles at the call today if enough people are online

Reply:
We can give/get users roles via the [Auth0 Management API](https://auth0.com/docs/api/management/v2#!/Users/get_user_roles). 

The access token used to make requests from the front-end will contain the users roles (scope) in its payload.